site/public/js/sql-toolbox.js

```diff
@@ -73,3 +74,33 @@ async function runSqlQuery() {
         console.error(exc);
     }
 }
+
+async function downloadSqlResult(){
+    const results = document.getElementById('query-results');
+    let i;
+    let j;
```

Comment:
Move these two declarations into the for loop, putting it outside is bit confusing IMO.